charts/kube-state-metrics/values.yaml

```diff
@@ -0,0 +1,161 @@
+# Default values for kube-state-metrics.
+prometheusScrape: true
+image:
+  repository: quay.io/coreos/kube-state-metrics
```

Comment:
Any chance we can change this to the gcr image and build a 1.9.7 image there?

See https://github.com/prometheus-community/helm-charts/issues/373

Reply:
@eddiezane Good idea. Though based on past experience I would prefer that to be a follow-up PR to keep the scope clear in this one. Does that sound reasonable?

Reply:
1.9.7 did not have the gcr images yet, but you can do the migration in v2.0.0 to gcr.

Reply:
@scottrigby that works for me.

@lilic Since v2.0.0 will have breaking changes I'd like to advocate for the last v1 release to have an official image (with multi arch support) cut.